### ISSUE TITLE
How to cancel async upload request

### ISSUE THREAD

**Issue by OpticalEngineering** [OneDrive/onedrive-api-docs#1759]
how to cancel async upload request ?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 42443681-dd86-ce0c-587f-2336f965ebaf
* Version Independent ID: abcbaeba-560b-661d-0127-5c44d9089915
* Content: [Upload from a URL - OneDrive API - OneDrive dev center](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_upload_url?view=odsp-graph-online)
* Content Source: [docs/rest-api/api/driveitem_upload_url.md](https://github.com/OneDrive/onedrive-api-docs/blob/live/docs/rest-api/api/driveitem_upload_url.md)
* Service: **one-drive**
* GitHub Login: @JeremyKelley
* Microsoft Alias: **JeremyKe**

**Comment by ificator** [comment]
I don't believe there is a way to cancel it currently... can you explain your scenario? Did you start an "upload from url" on a large file and it's taking too long so you want to stop it?

Ordinarily the uploads are pretty quick and so the window in which a cancel would be useful is pretty small.

**Comment by Ddaranee19** [comment]
> จะยกเลิกคำขออัพโหลด async ได้อย่างไร?
> 
> #### รายละเอียดเอกสาร
> ⚠ _ห้ามแก้ไขส่วนนี้ จำเป็นสำหรับการเชื่อมโยงปัญหา learn.microsoft.com ➟ GitHub_
> 
> * รหัส: 42443681-dd86-ce0c-587f-2336f965ebaf
> * รหัสอิสระเวอร์ชัน: abcbaeba-560b-661d-0127-5c44d9089915
> * เนื้อหา: [อัปโหลดจาก URL - OneDrive API - ศูนย์นักพัฒนา OneDrive](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_upload_url?view=odsp-graph-online)
> * แหล่งที่มาของเนื้อหา: [docs/rest-api/api/driveitem_upload_url.md](https://github.com/OneDrive/onedrive-api-docs/blob/live/docs/rest-api/api/driveitem_upload_url.md)
> * บริการ: **ไดรฟ์เดียว**
> * เข้าสู่ระบบ GitHub:@JeremyKelley
> * นามแฝงของ Microsoft: **JeremyKe**

>*https://youtube.com/@peerapolchularat1442?si=tOnzCddPaKRmOH97

**Comment by OpticalEngineering** [comment]
I uploaded a large number of files through upload from url. Now there is a file that needs to be uploaded immediately, but I need to wait for the previous file to be uploaded. I want to cancel files that have not been uploaded successfully so that new files can be uploaded immediately
